includes/ui/tabs/overview.php

```diff
@@ -7,14 +7,12 @@
 
 defined( '\\ABSPATH' ) || exit;
 
-$plugin = redis_object_cache();
```

Comment:
Why is this `$roc` and not `$plugin`?

Reply:
`$plugin` is considered a global WordPress variable and should therefore not be overwritten. As we work with namespaces this should not be a problem but I guess it is best practice to avoid naming variables after WordPress globals.